### PR TITLE
Jay branch

### PR DESCRIPTION
**Pull request by ArtfulJW**

### CODE REVIEW

File: public/css/login.css [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+}
+
+#Base-Container-OpacityCard{
+    z-index: 1;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    /* background-image: url(../images/mountain1.jpg); */
```
Comment (nok-ko) [inline public/css/login.css:54]:
```suggestion
```

File: public/css/login.css [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+    border: none;  
+    border-radius: 25px;
+    width: 70vw;
+    height: auto;
+}
+
+#Button-Login{
+    border: none;    
+    border-radius: 25px;
+    background-color: #FBB33B;
```
Comment (nok-ko) [inline public/css/login.css:97]: Might be worth putting commonly-used colours in [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

File: public/css/login.css [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+@font-face {
+    font-family: 'Satoshi-BlackItalic';
+    src: url('../fonts/Satoshi-BlackItalic.woff2') format('woff2'),
+        url('../fonts/Satoshi-BlackItalic.woff') format('woff'),
+        url('../fonts/Satoshi-BlackItalic.ttf') format('truetype');
+    font-weight: 900;
+    font-display: swap;
+    font-style: italic;
+}
```
Comment (nok-ko) [inline public/css/login.css:261]: Maybe put this in a separate fonts CSS file for readability.

File: public/css/signup.css [hunk 4]
```diff
@@ -0,0 +1,278 @@
+body {
+    background-color: #dfdfdf;
+}
+
+/* Universal Selector */
+* {
+
```
Comment (nok-ko) [inline public/css/signup.css:7]: There's a bunch of duplicate code between `login.css` and `signup.css`, maybe run them through a diff tool and factor out the common lines into a `common.css` file?

File: public/css/signup.css [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+@font-face {
+    font-family: 'Satoshi-BlackItalic';
+    src: url('../fonts/Satoshi-BlackItalic.woff2') format('woff2'),
+        url('../fonts/Satoshi-BlackItalic.woff') format('woff'),
+        url('../fonts/Satoshi-BlackItalic.ttf') format('truetype');
+    font-weight: 900;
+    font-display: swap;
+    font-style: italic;
+}
```
Comment (nok-ko) [inline public/css/signup.css:278]: Seriously…

File: README.txt [hunk 6]
```diff
@@ -1 +1,2 @@
-test
\ No newline at end of file
+test
+JayBranch Edit 1
```
Comment (nok-ko) [inline README.txt:2]: Maybe just delete the `README`, it's all example changes…

File: public/js/client.js [hunk 7]
```diff
@@ -54,7 +54,11 @@ ready(function(){
         xhr.send(params);
     }
 
+<<<<<<< HEAD
     // Detect #LOGIN button click, Make POST request TO THE SERVER
+=======
+    // POST TO THE SERVER
+>>>>>>> 701adf1 (First Push)
```
Comment (nok-ko) [inline public/js/client.js]: Whoops, forgot to resolve this conflict.

```suggestion
    // Detect #LOGIN button click, Make POST request TO THE SERVER
```

File: public/js/client.js [hunk 8]
```diff
@@ -78,4 +82,11 @@ ready(function(){
         }, queryString);
     });
 
+<<<<<<< HEAD
+=======
+    ajaxPOST("/marketplace", function(data){
+        //document.getElementById("grid-item-pokeMartTable").innerHTML = ;
+    });
+
+>>>>>>> 701adf1 (First Push)
```
Comment (nok-ko) [inline public/js/client.js]: Same here.
```suggestion
```

**Review by nok-ko — CHANGES_REQUESTED** [review]
Maybe remove the empty `style.css` file if you're not using it, or put common styles in there.

Could ask @speakingoftime for an SVG version of the logo instead of a PNG one? Doesn't really matter though.

What's the copyright situation on those pictures of mountains and stuff, where did you get them from?

File: html/signup.html [hunk 9]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>ShockGuard - Sign up</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/img/favicon.ico" rel="icon" type="image/x-icon" />
+    <link href="../public/css/signup.css" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (speakingoftime) [reply html/signup.html:9]: There wasn't a webfonts version of Satoshi that I could find and figured we could just download and add the fonts to our project

The fonts came with an end user license agreement (Fontshare EULA.txt) as well as a separate css file that had a block comment with the copyright info as well (satoshi.css)

We could copy the **Fontshare EULA.txt** file into public/fonts and **satoshi.css** to public/css

The Google Font link to Monserrat file should be deleted though

File: html/login.html [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+  </head>
+  <body>
+
+    <div id="Base-Container">
+        <div id="Base-Container-Image"></div>
+        <div id="Base-Container-OpacityCard"></div>
+        <div id="Form-Container">
+            <form id="grid-container-form">
+                <img src="../public/images/ShakeGuard.png" alt="" class="ShakeGuard-Logo">
+                <input class="form-item Grid-Container-Form-Input" id="email" type="text" required="required" placeholder="Username" value=""/>
```
[the remark this answers is not in the document]
Comment (ArtfulJW) [reply html/login.html:19]: Fixed the naming convention.

File: public/css/login.css [hunk 11]
```diff
@@ -0,0 +1,261 @@
+body {
+    background-color: #dfdfdf;
+}
+
+/* Universal Selector */
+* {
+    
+}
+
```
[the remark this answers is not in the document]
Comment (ArtfulJW) [reply public/css/login.css:9]: Removed it

File: public/css/login.css (hunk 3 above)
Comment (speakingoftime) [reply public/css/login.css:261]: There was actually a separate css file that came with the web fonts.

Once that "satoshi.css" file is added to the public/css folder, we could use
`@import url('satoshi.css') `

...I think...

File: html/signup.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (ArtfulJW) [reply html/signup.html:9]: Removed link to the Google WebFonts.

File: public/css/login.css [hunk 12]
```diff
@@ -0,0 +1,261 @@
...
+    color: #FFFFFF;
+}
+
+a:visited{
+    color: #FFFFFF;
+}
+
+.ShakeGuard-Logo{
+    margin-top: 5vh;
+
```
[the remark this answers is not in the document]
Comment (ArtfulJW) [reply public/css/login.css:20]: Removed erroneous spacing

File: public/css/login.css (hunk 1 above)
Comment (ArtfulJW) [reply public/css/login.css:54]: Removed commented out code

File: README.txt (hunk 6 above)
Comment (ArtfulJW) [reply README.txt:2]: Deleted

Comment (nok-ko) [reply README.txt:2]: Remember to commit + push your changes once you've made them, then once it's ready for review again, request review on this PR. 👍

**Review by nok-ko — APPROVED** [review]

**Review by nok-ko — CHANGES_REQUESTED** [review]
Whoops, approved by accident. Resolve the previous comments, ping me when you're done :)

File: public/css/signup.css (hunk 4 above)
Comment (ArtfulJW) [reply public/css/signup.css:7]: Used style.css for the common code.

File: public/fonts/satoshi.css [hunk 13]
```diff
@@ -0,0 +1,178 @@
+/**
+ * @license
+ *
+ * Font Family: Satoshi
+ * Designed by: Deni Anggara
+ * URL: https://www.fontshare.com/fonts/satoshi
+ * © 2022 Indian Type Foundry
```
Comment (nok-ko) [inline public/fonts/satoshi.css:7]: Still have some concerns about this…

image

**Review by nok-ko — COMMENTED** [review]
LGTM, except I'm not sure if we're abiding by the font license. Ech. @speakingoftime, any input? Their [license](https://www.fontshare.com/terms#limitations-of-usage) says we can't host the fonts on a public server, which GitHub is.

File: public/fonts/satoshi.css (hunk 13 above)
Comment (nok-ko) [reply public/fonts/satoshi.css:7]: https://www.fontshare.com/terms#limitations-of-usage

File: public/css/signup.css (hunk 5 above)
Comment (speakingoftime) [reply public/css/signup.css]: please delete the @face stuff and just paste this in signup.css

`@import url('https://api.fontshare.com/css?f[]=satoshi@400,401,700,701&display=swap');
@import url('https://api.fontshare.com/css?f[]=sharpie@400,700&display=swap');
`

whenever you need to use the font Satoshi or Sharpie
Satoshi:
`font-family: 'Satoshi', sans-serif;`

Sharpie:
`font-family: 'Sharpie', serif;`

Font weight regular:
`font-weight: 400;`

Font weight bold
`font-weight: 700;`

File: public/fonts/satoshi.css (hunk 13 above)
Comment (speakingoftime) [reply public/fonts/satoshi.css:7]: we can use the fonts through the FontShare API which realized was available so we can delete the font files on our end altogether, so we're not sharing the font files

Comment (ArtfulJW) [reply public/fonts/satoshi.css]: Deleted 'font' folder.
Imported API for Font in style.css

**Review by nok-ko — APPROVED** [review]
LGTM!

**Pull request merged**